README.md

```diff
@@ -15,12 +15,14 @@ The configuration requires following properties:
 - `port` - integer (required): Server port (default port is 21)
 - `username` - string (required): User with correct access rights
 - `password` - string (optional): Password for given User
-- `path` - string (required): Path to specific file or glob syntax path (FTP server must support recursive listing)
+- `path` - string (required): Path to specific file or glob syntax path
     - FTP(s) uses absolute path
     - SFTP uses relative path according to user's HOME directory
 - `connectionType` - string (required): Type of connection (possible value [FTP|FTPS|SFTP])
 - `privateKey` - string (optional): Possible to use only with SFTP connectionType.
 - `onlyNewFiles` - boolean (optional): Compares timestamp of files from last run and download only new files
+- `recurseManually` - boolean (optional, default: false): Open each directory instead of using recursive listing. Use
```

Comment:
`manualRecursion`? nebo `listing: recursive/manual`?

Reply:
Ono to je asi přímo podle toho jak se to používá v tom adapteru: https://github.com/thephpleague/flysystem/blob/master/src/Adapter/Ftp.php#L30

Reply:
jo s tím překladem jsem taky trochu bojoval, ale říkal jsem si, že to kluci vymysleli líp. Dám tam `manualRecursion`. To se mi líbí víc :+1: